src/pages/user_profile/DAAs.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { DAA } from '../../libs/ajax/DAA';
+
+export default function DAAs(props) {
+
+  const {
+    issuedOn,
+    issuedBy,
+    daas
+  } = props;
+
+  const DAADownload = (id, fileName) => {
+    return (
+      <div className="flex flex-row" style={{ justifyContent: 'flex-start', marginBottom: '30px'}}>
+        <div>
+          <a target="_blank" rel="noreferrer" onClick={() => DAA.getDaaFileById(id, fileName)} className="button button-white" style={{ marginRight: '2rem', display: 'flex', alignItems: 'center'}}>
+            <span className="glyphicon glyphicon-download"></span>
```

Comment:
I think this could use a slight amount of padding on the right, see the lower button vs the upper button, that's a 1 rem space:
![Screenshot 2024-07-12 at 9 51 33 AM](https://github.com/user-attachments/assets/0e156fa4-adf1-48f4-88d7-66ddee20dda5)